### PR TITLE
Another major change to UI theme

### PR DESCRIPTION
**Pull request by kimly888**
- Add fireworks on button click
- Add transition for header

### CODE REVIEW

**Review by mutokrm — COMMENTED** [review]
Thank you for your concise commit messages! I'll merge this.

File: client/src/components/Main.jsx [hunk 1]
```diff
@@ -1,7 +1,16 @@
 import "./Main.css";
 import KanjiCard from "./KanjiCard";
+import { Fireworks } from "fireworks/lib/react";
```
Comment (mutokrm) [inline client/src/components/Main.jsx:3]: Is this path correct? I couldn't find this file in this repo.

Comment (kimly888) [reply client/src/components/Main.jsx:3]: sorry you will need to do an npm install for this library. 
`npm install fireworks --save`

Comment (mutokrm) [reply client/src/components/Main.jsx:3]: I see! Let me try again!

Comment (mutokrm) [reply client/src/components/Main.jsx:3]: Sorry, since I reverted this PR, can you make a PR again? I'm sorry!
